flixel/addons/display/FlxClothSprite.hx

```diff
@@ -0,0 +1,533 @@
+package flixel.addons.display;
+import flixel.FlxCamera;
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.graphics.FlxGraphic;
+import flixel.graphics.frames.FlxFrame;
+import flixel.graphics.frames.FlxFrame.FlxFrameAngle;
+import flixel.graphics.frames.FlxFrame.FlxFrameType;
+import flixel.math.FlxPoint;
+import flixel.math.FlxRect;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+import flixel.util.FlxDestroyUtil;
+import flixel.util.FlxSpriteUtil;
+import openfl.display.BitmapData;
+import openfl.display.Graphics;
+import openfl.geom.Point;
+import openfl.Vector;
+
+@:keep @:bitmap("assets/images/logo/default.png")
+private class GraphicDefault extends BitmapData {}
```

Comment:
Shouldn't include this again here, pretty sure that duplicates the graphic in memory. You can use `@:access` to access the original one.